internal/buildscripts/packaging/fpm/common.sh

```diff
@@ -77,7 +77,7 @@ download_smart_agent() {
     local dl_url=""
 
     if [ "$tag" = "latest" ]; then
-        tag=$( curl -sL "$SMART_AGENT_RELEASE_URL/latest" | jq -r '.tag_name' )
+        tag=$( curl -sfL "$SMART_AGENT_RELEASE_URL/latest" | jq -r '.tag_name' )
         if [ -z "$tag" ]; then
             echo "Failed to get tag_name for latest release from $SMART_AGENT_RELEASE_URL/latest" >&2
             exit 1
```

Comment:
For a few of these updated curl commands, there's a check directly after to see if the result of the curl command has any contents (`if [ -z "$tag" ]; then`). Is that a valid check for success? If it is, it may be a clearer error message than the script failure. If it's an equivalent check, we can probably remove `-f` when we check the result in the script.

Reply:
Perfect, thanks for clarifying! Definitely stick with the `-f` then.